caffe2/serialize/inline_container.cc

```diff
@@ -276,8 +261,8 @@ size_t getPadding(
   // zip extra encoding (key, size_of_extra_bytes)
   padding_buf[0] = 'F';
   padding_buf[1] = 'B';
-  padding_buf[2] = (uint8_t)padding_size;
-  padding_buf[3] = (uint8_t)(padding_size >> 8);
+  padding_buf[2] = (char)padding_size;
```

Comment:
Why is this better?

Reply:
Because padding_buf is char array. ```(uint8_t)padding_size;``` has an implicit cast to char.